spring-cloud-gcp-pubsub/src/main/java/com/google/cloud/spring/pubsub/core/PubSubEventSpecificProperties.java

```diff
@@ -0,0 +1,458 @@
+/*
+ * Copyright 2021-2021 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.google.cloud.spring.pubsub.core;
+
+import com.google.api.gax.batching.FlowController.LimitExceededBehavior;
+
+/**
+ * Properties for Publisher or Subscriber specific configurations.
+ *
+ * @author João André Martins
+ * @author Mike Eltsufin
+ * @author Chengyuan Zhao
+ */
+public class PubSubEventSpecificProperties {
```

Comment:
What does "event specific" mean? Does it make sense to just call it `PubSubConfiguration`?

Reply:
+1 for simpler name.

Reply:
That's a much better name, thank you.